src/internals/selection.js

```diff
@@ -164,18 +166,22 @@ extend(ChartInternal.prototype, {
 					selecter = `.${selecter}${$$.getTargetSelectorSuffix(d.id)}`;
 				}
 
-				$$.main.selectAll(selecter)
-					.selectAll(`.${CLASS.shape}`)
+				$$.main.selectAll(`${selecter}`)
+					.selectAll(`circle.${CLASS.shape}`)
```

Comment:
Is to narrow the selector boundary, because `selectable` elements are &lt;circle> elements.